internal/featureflags/temporary.go

```diff
@@ -81,3 +81,12 @@ func GenerateJSONSchemas() FeatureFlag {
 		defaultEnabled: true,
 	}
 }
+
+// DashboardShareSettings toggles whether the dashboard share settings are downloaded and / or deployed.
+// Introduced: 2024-02-29; v2.12.0
+func DashboardShareSettings() FeatureFlag {
```

Comment:
Instead of introducing new one, I would suggest to go with Experimental (from `permanent.go`)

Reply:
Did you discuss how to continue on that? I think it is also used already for diffent apis right?
my only concern is, that at some point you don't really know what you are enabling exactly with that general FF :D

Reply:
That was my concern - `Experimental` doesn't say what it is enabling. I would stick with `MONACO_FEAT_DASHBOARD_SHARE_SETTINGS` for this feature as it is already there. Future ones we can decide in the refinement phase.